### DISCUSSION TITLE
Issues on connecting srsRAN's gNB  to OAI's FlexRIC (Dockerized)

### DISCUSSION THREAD

**Discussion by Ali78k** [srsran/srsRAN_Project#378] [General Help] [answered]
Hello, Community, I am interested in the srsRAN Project and O-RAN concept. A few weeks ago, thanks to the [excellent instruction](https://docs.srsran.com/projects/project/en/latest/tutorials/source/flexric/source/index.html), I set up srsRAN gNB and srs4g UE along with FlexRIC (near-RT RIC).

Now I want to prepare the same things - but dockerized. Following the instruction, I have written Dockerfile for Flexric (attached) and have edited the available docker/docker-compose.yml file (also attached). When I run gNB with the attached configuration, it can not connected to the Flexric. I saw the "Abort" SCTP response from the Flexric container in Wireshark.

![image](https://github.com/srsran/srsRAN_Project/assets/101015446/5442c595-34a2-4e36-9d2a-95462dbc2220)

Any hint will be appreciated.

[initabort.zip](https://github.com/srsran/srsRAN_Project/files/13725809/initabort.zip)
[docker-compose.yml.txt](https://github.com/srsran/srsRAN_Project/files/13725843/docker-compose.yml.txt)
[Dockerfile.txt](https://github.com/srsran/srsRAN_Project/files/13725844/Dockerfile.txt)
[flexgnb.yaml.txt](https://github.com/srsran/srsRAN_Project/files/13725845/flexgnb.yaml.txt)

**Comment by pgawlowicz** [comment, marked as answer]
please check this comment and try setting `bind_addr` for E2 in the gnb config
https://docs.srsran.com/projects/project/en/latest/tutorials/source/flexric/source/index.html#ric-running-on-a-different-machine

**Comment by hviet2603** [comment]
Hi, maybe it's a little bit late here, but for me, I had to configure the bind address of FlexRIC in flexric.conf inside the FlexRIC container. Normally FlexRIC uses the default config locates at /usr/local/etc/flexric/flexric.conf.
